ios/FluentUI/Table View/TableViewHeaderFooterView.swift

```diff
@@ -539,22 +554,10 @@ private class TableViewHeaderFooterTitleView: UITextView {
         self.textContainer.lineFragmentPadding = 0
         textContainerInset = .zero
         layoutManager.usesFontLeading = false
-        updateLinkTextColor()
-
-        NotificationCenter.default.addObserver(self,
```

Comment:
ooc - why don't we need this anymore?

Reply:
This view was independently observing color changes, but it's a private subview of its parent. I updated it so that the parent `TableHeaderFooterView` would just inject the current `fluentTheme` during updates